R/derive_params_growth_height.R

```diff
@@ -263,7 +281,12 @@ derive_params_growth_height <- function(dataset,
       by = c("sex_join", "heightu_join"),
       relationship = "many-to-many"
     ) %>%
-    filter(prev_height <= {{ height }} & {{ height }} < next_height)
+    mutate(ht_diff := abs(meta_height - {{ height }})) %>%
+    group_by(!!!by_vars) %>%
+    mutate(is_lowest = ht_diff == min(ht_diff)) %>%
+    group_by(!!!by_vars, is_lowest) %>%
+    filter(is_lowest & row_number() == 1) %>%
```

Comment:
Same as in `derive_params_growth_age()`, something weird happening here, we are loosing many records in ADVS template. What is the aim of having row_number == 1 here ?